config/routes.rb

```diff
@@ -1,4 +1,12 @@
 Rails.application.routes.draw do
+  post 'referral_agencies/create', to: 'referral_agencies#create'
```

Comment:
you can use the :resources keyword with an :except for the methods you don't need

https://guides.rubyonrails.org/routing.html#resources-on-the-web